### ISSUE TITLE
Management of user-defined analysis profiles (drymass.cfg)

### ISSUE THREAD

**Issue by gcojoc** [RI-imaging/DryMass#20]
related with #19 
- when created the 'main_location\dirXY\dirXY.zip_dm' directory, a drymass.cfg file should be placed inside. The drymass.cfg file should contain the same RI of the medium, pixel size, wavelength, sample size +- deviation, eccentricity +- deviation and name of background correction image (SID PHA bg.tif) for all directories in the main folder.  An option can be that a drymass.cfg file is manually placed by the user at the 'main_location' or the software ask in the beginning for these imput parameters. #19

**Comment by paulmueller** [comment]
It might be useful to also handle configuration files centrally, i.e.

```
dm_profile add profile_name /path/to/drymass.cfg  # add a profile to drymass
dm_profile export /path/to/export folder/  # will create the file /path/to/export_folder/profile_name.cfg
dm_analyze_sphere --profile profile_name /path/to/measurement  # will use `profile_name` to analyze data
```

TODO:
- [ ] implement command line script `dm_profile` with add / remove / export commands
- [ ] implement `--profile` or `-p` command line parameter in dialog.py

**Comment by paulmueller** [comment]
This is implemented in version 0.8.0. I also added a section in the docs: https://drymass.readthedocs.io/en/0.8.0/sec_gs_command_line_interface.html#advanced-usage
